identity-apps-core/apps/authentication-portal/src/main/webapp/includes/theme-skeleton.jsp

```diff
@@ -1318,6 +1318,11 @@
         color: var(--asg-input-field-base-label-text-color);
     }
 
+    .ui.checkbox:not(.toggle, .radio) input[type="checkbox"]:checked~label:after {
```

Comment:
Any specific reason to exclude radio and toggle inputs? Afaiu, branding changes should be reflected im those inputs too, shouldn't it? Please correct me if I'm wrong.

Reply:
AFAIK, authentication and recovery portals typically do not include toggles and radios. If, however, such elements are present, the color handling should be adjusted accordingly. This is because toggle and radio elements have differences compared to checkboxes. The specificity of the checkbox is likely specified for this reason. Refer this code https://github.com/wso2/identity-apps/blob/028e15f359362eddbdf9f635bbc7dfdb251dcc47/modules/theme/src/themes/default/modules/checkbox.overrides#L30

Reply:
If we cannot modify the styles to support toggle and radio buttons with this PR because it's OOS, let's create a separate issue to track it as an improvement.

Reply:
Yeah sure, created a issue to track this https://github.com/wso2/product-is/issues/18739